game/scripts/npc/abilities/faceless_void_time_lock.txt

```diff
@@ -34,12 +34,12 @@
       "03"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "chance_pct"                                      "10 15 20 25 25 25" //OAA, for picking screen tooltip
+        "chance_pct"                                      "15 20 25 30 35 40" //OAA, for picking screen tooltip
       }
       "04"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "bonus_damage"                                    "75 100 125 150 325 500"  //OAA, for picking screen tooltip
+        "bonus_damage"                                    "100 125 150 175 350 550"  //OAA, for picking screen tooltip
```

Comment:
if you do double the duration of stun I would just revert these changes (not additional nerfs needed)